### ISSUE TITLE
Setting a canonical URL in a post header doesn't override the old canonical URL

### ISSUE THREAD

**Issue by adam-arold** [TryGhost/Ghost#10367]
I tried setting a canonical url for a post in Code Injection > Post Header but I still see the old canonical URL set by Ghost.

I’m trying to publish a guest post on a blog and I can’t do it while I can’t set the canonical url for my post to point to the original article.

Is there a solution for this which works **now**? We’re blocked by this bug.

**Comment by ErisDS** [comment]
Hey @adam-arold  👋 We ask that you please do not use GitHub for help or support, the default issue template pointed you to our [forum](https://forum.ghost.org) for this type of question 😄 We use GitHub solely for bug-tracking and on-going feature development so we try to keep it noise free.

Many questions can be answered by reviewing our docs for [self-hosters](https://docs.ghost.org), our [theme API](https://themes.ghost.org), or our [public API](https://api.ghost.org). If you can't find an answer then our [forum](https://forum.ghost.org) is a great place to get community support, plus it helps create a central location for searching problems/solutions.

FYI: Many projects have their own support guidelines and GitHub will highlight them for you, or the project owners will use issue templates to point you in the right direction, please read them before opening issues

**Comment by adam-arold** [comment]
I tried the forum, but they said there is no solution. This is definitely a bug since I set a canonical url and they are duplicated.

**Comment by ahawkins** [comment]
I also have the same issue. I don't understand why this issue was closed. This seems like an acceptable work around to the glaring issue that Ghost does not allow users to set canonical URL's themselves.

**Comment by adam-arold** [comment]
Don't worry I'm opening another issue.
